projects/schematics/src/migrations/5_0/constructor-deprecations/data/shipping-address.component.migration.ts

```diff
@@ -1,49 +0,0 @@
-import {
-  ACTIVATED_ROUTE,
-  ACTIVE_CART_SERVICE,
-  ANGULAR_ROUTER,
-  CHECKOUT_DELIVERY_SERVICE,
-  CHECKOUT_STEP_SERVICE,
-  GLOBAL_MESSAGE_SERVICE,
-  SHIPPING_ADDRESS_COMPONENT,
-  SPARTACUS_CORE,
-  SPARTACUS_STOREFRONTLIB,
-  TRANSLATION_SERVICE,
-  USER_ADDRESS_SERVICE,
-} from '../../../../shared/constants';
-import { ConstructorDeprecation } from '../../../../shared/utils/file-utils';
-
-export const SHIPPING_ADDRESS_COMPONENT_MIGRATION: ConstructorDeprecation = {
```

Comment:
not included in migration, which is GOOD

because it was moved and renamed to CheckoutDeliveryAddressComponent, which the script won't be able to 'detect'


Just leaving it here for a reminder for a list for me in checkout